spec/README.md

```diff
@@ -123,3 +123,17 @@ All other names in these categories are reserved for the use of implementations
 > [!NOTE]
 > Users defining custom names SHOULD include at least one character outside these ranges
 > to ensure that they will be compatible with future versions of this specification.
+
```

Comment:
Reverse the order of these, so that the "yes it is backwards compatible" message comes first.

Reply:
I incorped your suggestions, but think they might be saying the wrong thing.

The data model is intended (I think) to be "forwards compatible" from the point of view that a valid data model representation of this version will still be valid later.

The data mode is intended (I think) to be _potentially_ non-backwards compatible from the point of view that a future valid data model representation might not be readable by current data model implementations (because it uses a new feature).

Future version of the syntax are guaranteed to be compatible with the current data model, but the results might be different from the future data model's representation.

Reply:
I made a set of changes based on the above.

Reply:
LGTM now